interview-questions.md

```diff
@@ -7,44 +7,79 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. As a developer, I am creating a Rails application with a model called Cohort that has_many students, but OOPS! I forgot to add the foreign key. How can I fix this mistake? What is the name of the foreign key? Would the foreign key be on the Cohort model or the Student model?
 
-  Your answer:
+  Your answer: 
+  As far as fixing the mistake, I will have to do research, but I am certain I know where to look. The name of the foreign key would be student_id. Student_id, the foreign key would belong to the Student model
 
   Researched answer:
-
+If you forget to add the foreign key when creating a model, you can use migrations to fix the problem. According to something I found on Stack Overflow, you can use $rake: db:rollback then add_column and in this case enter "student_id: integer". The foreign key will always be on the model that "belongs_to" so in this case, "student_id" would belong to the Student model. A time I used a foreign key was on the Wildlife Tracker Challenge when I created the Sighting model.
 
```

Comment:
Rollback could be an approach, however, rollback does have some limitations.  Another approach is to simply add a migration and update the database in that way.